src/Deferred.php

```diff
@@ -14,6 +14,9 @@ public function __construct(callable $canceller = null)
         $this->canceller = $canceller;
     }
 
+    /**
+     * @inheritdoc
+     */
```

Comment:
Oh, you right. I have a specific project where most of functions/methods don`t contain body (for security reason). So phpdoc was the only way to view returned value. Sorry for bothering you, I am going to close this PR.